src/main/java/gregtech/common/covers/ender/CoverAbstractEnderLink.java

```diff
@@ -97,6 +98,11 @@ protected final UUID getOwner() {
         return isPrivate ? playerUUID : null;
     }
 
+    protected boolean canAccess(UUID other) {
+        return this.playerUUID == null || this.playerUUID.equals(other) ||
+                FTBTeamHelper.isSameTeam(this.playerUUID, other);
+    }
+
```

Comment:
Is the KeyPressed Override in this class an option to move to our GTTextField, and then just handle behavior based on if a sync handler is passed in to the method?